### DISCUSSION TITLE
is this happening to anyone else when clicking to see more images?

### DISCUSSION THREAD

**Discussion by sabi3293043** [civitai/civitai#289] [Q&A]
when you click more images of a model all images dont show up. i know that beacause it is happening for all the models i uploaded and i know i added more sample images.

i dont know if this is already known. i searched for it in discussions nothing came up, maybe my wording was wrong.

**Comment by JohnMadden123456** [comment]
May it be this problem (already fixed)? https://github.com/civitai/civitai/issues/287
